### PR TITLE
Update whitepaper.md - Cordeta

### PR DESCRIPTION
**Pull request by cordeta**
From Line 1 to 61 A total of 1112 words. (Source codes and variables were not counted)

Thanks

### CODE REVIEW

**Review by codygee237 — COMMENTED** [review]
Your translation: Gayun din
Suggested translation: Gayundin

**Comment by cordeta** [comment]
Thanks bro ;)

File: tl-tl/node/whitepaper.md [hunk 1]
```diff
@@ -1,64 +1,64 @@
...
+​Ang artikulong ito ay iminumungkahi ang isang pinahusay na  Byzantine na toleransiya sa kamalian na algoritmo, na binagay para sa sistemang blockchain. Gayundin, sa sistemang ito, ang mga mensahe ay sumailalim sa pagkawala, pagkasira, pagkawalang-kilos, at pag-uulit. Ganun din, ang pagkaka-ayos ng pag-papadala ay hindi kailangan na naaalinsunod sa pagkakaayos ng pagtanggap ng mga mensahe. Ang mga aktibidad ng mga noda ay maaring maging arbitraryo, sila ay pwedeng sumali at umalis sa network sa kahit na anong panahon; sila din ay pwedeng magbasura at pumeke ng impormasyon o sadyang titigil lang sa pagtatrabaho. Artipisyal o hindi artipisyal na pagkakamali ay pwede rin mangyari. Ang ating algoritmo ay nagbibigay din ng 𝑓 = ⌊ (𝑛−1) / 3 ⌋ na toleransiya ng kamalian sa sistemang konsensus na binubuo ng n noda. Ang kapasidad ng toleransiya nito ay sumasaklaw ng seguridad at sa abilidad na pwedeng gamitin na angkop sa kahit na anong na anumang sinasaklawang network.
```
Comment (severinolorillajr) [inline tl-tl/node/whitepaper.md]: *Ang artikulong ito ay nagmumungkahi ng isang pinahusay na (double space) Byzantine

File: tl-tl/node/whitepaper.md [hunk 2]
```diff
@@ -1,64 +1,64 @@
...
+Ang blockchain ay isang desentralisadong pagbabahagi ng sistema ng pagtatala. Ito ay pwedeng gamitin sa pagrerehistro o pag-iisyu ng makabagong digital na mga bagay na may halaga, mga sertipikong ikaw ay nagmamay-ari ng mga ari-arian, kreditong mga puntos at iba pa. Ito ay nagbibigay-daan din sa paglipat, pagbayad, at mga transaksyong tao-sa-taong pamamaraan. Ang teknolohiya ng blockchain ay orihinal na iminungkahi ni Satoshi Nakamoto sa paraan ng listahan ng liham na kriptograpiya, halimbawa, ang Bitcoin. Mula noon, sandamakmak na mga aplikasyon na batay sa blockchain ang lumitaw, tulad ng sistemang e-cash, palitan ng mga stock equity at ang mga sistemang smart na kontrata.
```
Comment (severinolorillajr) [inline tl-tl/node/whitepaper.md]: equity ng stock

Comment (severinolorillajr) [inline tl-tl/node/whitepaper.md]: Smart na Kontrata (capitalize)

File: tl-tl/node/whitepaper.md [hunk 3]
```diff
@@ -1,64 +1,64 @@
...
 
-A blockchain system is advantageous over a traditional centralized ledger system for its full-openness,immutability and anti-multiple-spend characters, and it does not rely on any kind of trusted third-party.
+Ang sistemang blockchain ay may kalamangan kesa sa tradisyunal na sentralisadong systema ng pagtatala dahil sa ito'y bukas ,sistemang kailan man ay hindi mababago at may kontra sa maramihang paggasta na mga karakter, at ito din ay hindi umaasa sa kahit na anumang ikatlong partid  na mapagkakatiwalaan.
```
Comment (severinolorillajr) [inline tl-tl/node/whitepaper.md]: bukas ,sistemang to bukas, sistemang

Comment (severinolorillajr) [reply tl-tl/node/whitepaper.md]: partid to partido

File: tl-tl/node/whitepaper.md [hunk 4]
```diff
@@ -1,64 +1,64 @@
...
 
-However, like all distributed systems, blockchain systems are challenged with network latency, transmission errors, software bugs, security loopholes and black-hat hacker threats. Moreover, its decentralized nature suggests that no participant of the system cannot be trusted. Malicious nodes may emerge, so does data difference due to conflicting interests.
+Gayunpaman, kayagaya sa ibang sistema ng pagbabahagi, ang mga sistemang blockchain ay hinahamon ng paghinda sa bilis ng network, mga kamalian sa transmisyon, mga softwer bugs, mga butas sa seguridad at ang mga banta ng black-hat hacker. Bukod pa rito, ang  desentralisadong katangian ay nagpapahiwatig na walang kalahok sa sistema ang mapagkakatiwalaan. May mga malisyong mga noda ang posibleng lumabas, gayundin ang pagkakaiba nga mga datos buhat ng mga magkakasalungat na mga interes.
```
Comment (severinolorillajr) [inline tl-tl/node/whitepaper.md]: softwer to software.
Not all transliterated words are acceptable.

File: tl-tl/node/whitepaper.md [hunk 5]
```diff
@@ -1,64 +1,64 @@
...
 
-Nodes, after monitoring the broadcasting and receiving the proposal, shall validate the transactions. They cannot write an illegal transaction in the memory once the latter is exposed. If an illegal transaction is contained in the proposal, this round of consensus will be abandoned and the View change will take place immediately. The validation procedures are as follows:
+Ang mga noda, matapos mamonitor ang pagpapahayag at pagtatanggap ng mga iminumungkahi, ang magpapatunay ng mga transaksyon. Hindi sila pwedeng sumulat ng illegal na transaksyon sa memorya nang sa huli ay mailantad. Kung ang illegal na transaksyon ay nakapaloob sa iminumungkahi , ang ikot ng konsensus na iyon ay iaabandona at ang pagbabago sa View ay magsisimula agad. Ang pamamaraan ng pagpapatunay ang mga sumusunod:
```
Comment (severinolorillajr) [inline tl-tl/node/whitepaper.md]: iminumungkahi to mungkahi

Comment (severinolorillajr) [reply tl-tl/node/whitepaper.md]: illegal to bawal
text<space>,</space>text to text,</space>test

File: tl-tl/node/whitepaper.md [hunk 6]
```diff
@@ -1,64 +1,64 @@
...
+6) Kahit na anumang noda, matapos matanggap ang buong bloke, ay buburahin ang transaksyon nasa katanungan na galing sa kanyang memorya at magsisimula ng panibagong pag-ikot para sa konsensus;
 
-It is required that, for all the consensus nodes, at least 𝑛 − 𝑓 nodes are in the same original state. This is to say, for all the nodes i, the block height h and View number v are the same. This is not difficult, consistency of h could be reached by synchronizing the blocks while consistency of v could reached by changing the View. Block synchronizing is not covered in this article. For View change, check next section.
+Kinakailangan na, sa lahat ng mga noda ng konsensus, at hindi bababa sa  𝑛 − 𝑓 ng mga noda ang nasa orihinal na estado. Ito ay para sabihinna para sa laht ng nodang i, ang taas ng blokeng h and bilang ng View na v ay magkapareha. Ito ay hindi mahirap, pagkakapareho-pareho ng h ay maabot sa paraang ng  pagsisingkronisa ng mga bloke habang ang pagkakapare-pareha ng  v ay maaabot sa paraan ng pagbabago ng View. Ang pagsisingkronisa ng bloke ay hindi  sakop sa artikulong ito. Para sa pagbabago ng view, tingnan ang susunod na seksyon.
```
Comment (severinolorillajr) [inline tl-tl/node/whitepaper.md]: sabihin na para sa lahat ng

Comment (severinolorillajr) [reply tl-tl/node/whitepaper.md]: paraang ng  pagsisingkronisa (double space)

Comment (severinolorillajr) [reply tl-tl/node/whitepaper.md]: hindi  sakop (double space)

File: tl-tl/node/whitepaper.md [hunk 7]
```diff
@@ -1,64 +1,64 @@
...
+3) Pagkatapos ng oras t, ang tagapagsalita ay magpapadala ng 〈𝑃𝑒𝑟𝑝𝑎𝑟𝑒𝑅𝑒𝑞𝑢𝑒𝑠𝑡,ℎ,𝑣,𝑝,𝑏𝑙𝑜𝑐𝑘,〈𝑏𝑙𝑜𝑐𝑘〉<sub>𝜎𝑝</sub>〉； 
 
-4) After receiving the proposal, congressmen i send 〈𝑃𝑒𝑟𝑝𝑎𝑟𝑒𝑅𝑒𝑠𝑝𝑜𝑛𝑠𝑒,ℎ,𝑣,𝑖,〈𝑏𝑙𝑜𝑐𝑘〉<sub>𝜎𝑖</sub>〉 ;
+4) Matapos matanggap ang lahat ng  iminumungkahi, ang mga kongressman i ay magpapadala ng 〈𝑃𝑒𝑟𝑝𝑎𝑟𝑒𝑅𝑒𝑠𝑝𝑜𝑛𝑠𝑒,ℎ,𝑣,𝑖,〈𝑏𝑙𝑜𝑐𝑘〉<sub>𝜎𝑖</sub>〉 ;
 
-​5) Any node, upon receiving at least 𝑛 − 𝑓  〈𝑏𝑙𝑜𝑐𝑘〉<sub>𝜎𝑖</sub>, reaches a consensus and publishes a full block;
+​5) Kahit na anumang noda, sa panahon ng pagtanggap ng hindi bababa sa 𝑛 − 𝑓  〈𝑏𝑙𝑜𝑐𝑘〉<sub>𝜎𝑖</sub>, ay aabot sa isang konsensus at maglalathala ng isang buong bloke;
 
-6) Any node, after receiving the full block, deletes the transaction in question from its memory and begins the next round the consensus;
+6) Kahit na anumang noda, matapos matanggap ang buong bloke, ay buburahin ang transaksyon nasa katanungan na galing sa kanyang memorya at magsisimula ng panibagong pag-ikot para sa konsensus;
```
Comment (severinolorillajr) [inline tl-tl/node/whitepaper.md]: transaksyong nasa

File: tl-tl/node/whitepaper.md [hunk 8]
```diff
@@ -1,64 +1,64 @@
...
+###  Pangkalahatang Pamamaraan 
 
-Set the time intervals of block generation as t, under normal circumstances, the algorithm executes in the following procedures：
+Itakda ang mga agwat ng oras ng paggawa ng bloke sa t, sa ilalim ng mga normal na pangyayari, ang algoritmo ay magpapatupad ng sumusunod na pamamaraan：
 
-1) A node broadcasts transaction data to the entire network, attached with the sender signature;
+1) Ang pagpapahayag ng transaktsyon na datos ng noda sa panghalahatang network , na nakalakip ang lagda ng nagpadala;
 
-2) All bookkeeping nodes monitors transaction data broadcasting independently and stores the data in its memory respectively; 
+2) Lahat ng mga noda na bookkeeping ay nagmomonitor ng mga datos ng transaksyon na namamahayag ng malaya at itinatago ang mga datos sa kanyang sariling memorya ayon sa pagkakabanggit;
```
Comment (severinolorillajr) [inline tl-tl/node/whitepaper.md]: node ng bookkeeping

File: tl-tl/node/whitepaper.md [hunk 9]
```diff
@@ -1,64 +1,64 @@
...
 
-​A new block will be generated with each round of consensus, with at least 𝑛 − 𝑓 signatures from bookkeeping nodes. Upon the generation of a block, a new round of consensus making shall begin, resetting v=0.
+​Isang bagong bloke ang mabubuo sa bawat pag-ikot ng konsensus, na hindi bababa sa 𝑛 − 𝑓 na mga lagda galing sa mga noda ng bookkeeping. sa panahon ng pagkakabuo ng bloke, isang bagong pag-ikot ng paggawa ng knsensus ang magsisimula, na-irereset sa v=0.
 
-###  General Procedures 
+###  Pangkalahatang Pamamaraan 
 
-Set the time intervals of block generation as t, under normal circumstances, the algorithm executes in the following procedures：
+Itakda ang mga agwat ng oras ng paggawa ng bloke sa t, sa ilalim ng mga normal na pangyayari, ang algoritmo ay magpapatupad ng sumusunod na pamamaraan：
```
Comment (severinolorillajr) [inline tl-tl/node/whitepaper.md]: bloke bilang t

File: tl-tl/node/whitepaper.md [hunk 10]
```diff
@@ -1,64 +1,64 @@
...
 
-All consensus nodes are required to maintain a state table to record current consensus status. The data set used for a consensus from its beginning to its end is called a View. If consensus cannot be reached within the current View, a View Change will be required. We identify each View with a number v, starting from 0 and it may increase till achieving the consensus.
+Lahat ng mga nodang para sa konsensus ay kailangang magpanatili ng talahanayang pang-estado para itala ang kasalukuyang estado ng konsensus. Ang set ng datos na ginamit para sa konsensus mula sa simula hanggang sa walas nito ay tinatawag na View. Kung ang konsensus hindi maabot ng kasalukuyang View, ang pagbabago ng View Change ang kinakailangan. Tutukuyin namin ang bawat isang View na may bilang v, na magsisimula sa 0 at ito ay maaaring madagdagan hanggang sa makamit ang konsensus.
```
Comment (severinolorillajr) [inline tl-tl/node/whitepaper.md]: grupo ng datos

File: tl-tl/node/whitepaper.md [hunk 11]
```diff
@@ -1,64 +1,64 @@
...
+​Ang ating algoritmo tinitiyak ang segurida at at abilidad na itoy pwedeng gamitin. Dahil sa mga maling noda sa konsensus ng paggawa ng hindi hihigit sa ⌊ (𝑛−1) / 3 ⌋ , ang panksyonalidad at katatagan ng sistema ay masisigurado. Sa loob nito, 𝑛 = |𝑅| ay nagmumungkahi na ang lahat ng kabuuang bilang ng noda na kasali sa paggagawa ng konsesus habang ang R ay ang ibig sabihin ay pangkat ng mga noda ng konsensus. Bigay ang 𝑓 = ⌊ (𝑛−1) / 3 ⌋ , f ay ang tumatayong kahulugan para sa pinakamataas na bilang ng mga maling noda na pwede sa sistema. Sa katunayan, ang kabuuang talaan ay binabantayan at iniingatan ng mga noda na bookkeeping habang ang mga ordinaryong noda ay hindi sumasali sa paggagawa ng konsensus. Ito ay para ipakita ang pangkalahatang pamamaraan ng paggawa ng konsensus.
```
Comment (severinolorillajr) [inline tl-tl/node/whitepaper.md]: Ang ating algoritmo ay nagtitiyak sa seguridad at pati na rin sa abilidad na ito'y pwedeng gamitin.

Comment (severinolorillajr) [reply tl-tl/node/whitepaper.md]: *masisigurado to masisiguro

File: tl-tl/node/whitepaper.md [hunk 12]
```diff
@@ -1,64 +1,64 @@
...
 
-Integrity and Authenticity of information transmission are ensured with cryptography while senders must attach signatures to the hash value of the message sent. Here we define 〈𝑚〉<sub>𝜎𝑖</sub> is the message m’s digital signature from node i, while D(m) is the hash value of message m. Without special clarification, all signature referred to in this article are signatures to the message hash value. 
+Itegridad at pagiging tunay ng mga transmisyon ng impormasyon ay natitiyak sa pamamagitan ng kriptograpiya habang ang nagpapadala ay kailangang maglagay ng lagda sa halaga ng hash ng mensaheng pinadala. Dito ay binibigyan nating ng kahulugan na ang 〈𝑚〉<sub>𝜎𝑖</sub> mensaheng m’s digital na lagda galing sa i na noda, habang ang D(m) ay ang halaga ng hash ng mensaheng m. Kahit walang espesyal na paglilinaw, lahat ng lagda na tinutukoy sa artikulong ito ay mga lagda sa halaga hash ng mensahe.
```
Comment (severinolorillajr) [inline tl-tl/node/whitepaper.md]: Ang Integridad at Pagpapatunay ng transmisyon ng impormasyon

File: tl-tl/node/whitepaper.md [hunk 13]
```diff
@@ -1,64 +1,64 @@
...
 
-Hypothetically, in this system, messages may subject to loss, damage, latency and repetition. Also, the sending order may not necessarily be consistent with the receiving order of messages. The activities of nodes could be arbitrary, they may join and quit the network at any time; they may also dump and falsify information or simply stop working. Artificial or non-artificial glitches may occur as well.
+Gayunpaman, sa sistemang ito,ang mga mensahe ay sumailalim sa pagkawala, pagkasira, pagkawalang-kilos, at pag-uulit. Ganun din, ang pagkaka-ayos ng pag-papadala ay hindi kailangan na naaalinsunod sa pagkakaayos ng pagtanggap ng mga mensahe. Ang mga aktibidad ng mga noda ay maaring maging arbitraryo, sila ay pwedeng sumali at umalis sa network sa kahit na anong panahon; sila din ay pwedeng magbasura at pumeke ng impormasyon o sadyang titigil lang sa pagtatrabaho. Artipisyal o hindi aryipisyal na mga glitches ay pwede din mangyari.
```
Comment (severinolorillajr) [inline tl-tl/node/whitepaper.md]: ito, ang

**Review by severinolorillajr — CHANGES_REQUESTED** [review]

**Comment by severinolorillajr** [comment]
@codygee237 , @cordeta 
Please have a thorough review next time as shown on the review that I had just made.
Thanks.

**Review by severinolorillajr — APPROVED** [review]

**Comment by cordeta** [comment]
I will sir :+1:  indeed a learning process for me too.

**Pull request merged**